### PR TITLE
Some maintenance tasks

### PR DESCRIPTION
**Pull request by swissspidy**
### Description

* Update `.editorconfig`
* Update plugin headers
* Remove unnecessary build tasks

### Type of change
Please select the relevant options:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (change which improves an existing feature. E.g., performance improvement, docs update, etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Acceptance criteria
- [x] My code follows WordPress coding standards.
- [x] I have performed a self-review of my own code.
- [x] If the changes are visual, I have cross browser / device tested.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added test instructions that prove my fix is effective or that my feature works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
-- Contributing: [Contributing Documentation Section](/docs/CONTRIBUTING.md)
-- Testing: [Testing Documentation Section](/docs/TESTING.md).
-
-
-## Changelog ##
-
-### 0.1.0 ###
-* In progress...
+# Core Sitemaps #
+**Contributors:** joemcgill,
```
Comment (joemcgill) [inline README.md]: Looks like the readme task stripped the list of contributors.

File: readme.txt [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-== Description ==
-
-See: https://make.wordpress.org/core/2019/06/12/xml-sitemaps-feature-project-proposal/
-
-== Changelog ==
-
-= 0.1.0 =
-* In progress...
+=== Core Sitemaps ===
+Contributors: joemcgill,
```
Comment (joemcgill) [inline readme.txt]: Probably needs to have the contributors added here.

**Review by joemcgill — CHANGES_REQUESTED** [review]
Looking pretty good. Can we include moving the contributor names to the readme.txt file in this change so they don't get stripped out completely?

Also, is removing the .pot file intentional as part of this change?

File: README.md (hunk 1 above)
Comment (swissspidy) [reply README.md]: Oh, looks like they were added here manually before. Will add them back properly.

**Comment by swissspidy** [comment]
@joemcgill Yes, that was intentional! WordPress.org handles string extraction. POT files are only really needed if you you plan on maintaining translations files yourself and not using WordPress.org. Also, it would require a `load_plugin_textdomain` call, which the plugin doesn't contain anyway.

**Review by joemcgill — APPROVED** [review]

**Comment by svandragt** [comment]
I feel that the Documentation section was a valuable part of the readme and should have been preserved.

**Comment by swissspidy** [comment]
Huh, that was by accident.

Sounds like we should remove that Grunt task though. Will create a new PR.

**Comment by swissspidy** [comment]
Created #114!

**Pull request merged**